app/javascript/controllers/child_vaccination_controller.js

```diff
@@ -27,12 +27,9 @@ export default class extends Controller {
     const child = json[childId];
 
     this.fullNameTarget.textContent = child["full_name"];
-    this.sexTarget.textContent = child["sex"];
     this.dobTarget.textContent = child["dob"];
-    this.consentTarget.textContent = child["consent"];
     this.gpTarget.textContent = child["gp"];
     this.nhsNumberTarget.textContent = child["nhs_number"];
-    this.screeningTarget.textContent = child["screening"];
```

Comment:
Interesting. I was removing fields also in the testing env PR, we should line these up.